setup.py

```diff
@@ -48,8 +48,11 @@ def get_long_description():
     keywords="taxii taxii2 server json cti cyber threat intelligence",
     packages=find_packages(exclude=["*.test", "*.test.data"]),
     install_requires=[
+        "appdirs>=1.4.4",
```

Comment:
This is chosen because 1.4.4 was the first release they made which was marked as stable. It's still seeing slow development but no releases since mid last year.